### PR TITLE
Update install log regexes configmap

### PR DESCRIPTION
**Pull request by LalatenduMohanty**

### CODE REVIEW

File: config/configmaps/install-log-regexes-configmap.yaml [hunk 1]
```diff
@@ -25,3 +25,8 @@ data:
       - "waiting for Kubernetes API: context deadline exceeded"
       installFailingReason: KubeAPIWaitTimeout
       installFailingMessage: Timeout waiting for the Kubernetes API to begin responding
+    - name: MonitoringOperatorError
+      searchRegexStrings:
+      - "failed to initialize the cluster: Cluster operator monitoring is still updating"
+      installFailingReason: monitoring operator fails to become ready
```
Comment (staebler) [inline config/configmaps/install-log-regexes-configmap.yaml]: The reason should be camel-case with no spaces.

See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties

Comment (dgoodwin) [reply config/configmaps/install-log-regexes-configmap.yaml]: We just had a race condition on this line. :)

**Pull request merged**